package.json

```diff
@@ -55,7 +55,6 @@
     "ember-cli-valid-component-name": "^1.0.0",
     "ember-cli-version-checker": "^1.3.1",
     "ember-router-generator": "^1.2.3",
-    "fs-extra": "^4.0.1",
```

Comment:
Even if we kept it, I think it should become a devDep anyways

Reply:
Agreed